src/main/java/de/qabel/desktop/ui/actionlog/ActionlogController.java

```diff
@@ -21,75 +34,175 @@
 
 public class ActionlogController extends AbstractController implements Initializable {
 
-
-	ResourceBundle resourceBundle;
 	List<ActionlogItemView> messageView = new LinkedList<>();
 
-
 	@FXML
 	VBox messages;
 
 	@FXML
 	ScrollPane scroller;
 
+	@FXML
+	TextArea textarea;
+
 	@Inject
 	ClientConfiguration clientConfiguration;
 
+	@Inject
+	private ContactRepository contactRepository;
+
+	Identity identity;
+	Contact c;
+	Date lastDate;
+	DropHTTP dHTTP = new DropHTTP();
 
 	public void initialize(URL location, ResourceBundle resources) {
 		try {
-			loadMessages();
-		} catch (EntityNotFoundExcepion entityNotFoundExcepion) {
+			identity = clientConfiguration.getSelectedIdentity();
+			c = new Contact(identity, identity.getAlias(), identity.getDropUrls(), identity.getEcPublicKey());
+			receiveDropMessages(null);
+
+			scroller.setVvalue(scroller.getVmax());
+			addChangeListener();
+
+		} catch (QblVersionMismatchException | QblDropInvalidMessageSizeException | QblSpoofedSenderException entityNotFoundExcepion) {
 			entityNotFoundExcepion.printStackTrace();
 		}
 	}
 
-	void loadMessages() throws EntityNotFoundExcepion {
-		messages.getChildren().clear();
-		createMessageItems();
-		if(scroller.getVmax() == scroller.getVvalue()) {
-			scroller.setVvalue(scroller.getVmax());
+	private void addChangeListener() {
+		((Region) scroller.getContent()).heightProperty().addListener((ov, old_val, new_val) -> {
+			if (scroller.getVvalue() != scroller.getVmax()) {
+				scroller.setVvalue(scroller.getVmax());
+			}
+		});
+	}
+
+	@FXML
+	protected void handleSubmitButtonAction(ActionEvent event) throws QblDropPayloadSizeException, EntityNotFoundExcepion {
```

Comment:
all these handlers could be easily tested, probably even the `changeListener` without a guiTest.